startevent/soap-cxf-server-start/src/test/java/com/camunda/bpm/example/spring/soap/start/BankAccountProcessTest.java

```diff
@@ -0,0 +1,90 @@
+package com.camunda.bpm.example.spring.soap.start;
+
+import static org.hamcrest.Matchers.is;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.camunda.bpm.example.spring.soap.start.v1.BankAccountServicePortType;
+import com.camunda.bpm.example.spring.soap.start.v1.BankRequestHeader;
+import com.camunda.bpm.example.spring.soap.start.v1.InvalidValueException;
+import com.camunda.bpm.example.spring.soap.start.v1.InvalidValueException_Exception;
+import com.camunda.bpm.example.spring.soap.start.v1.SetAccountNameRequest;
+import com.camunda.bpm.example.spring.soap.start.v1.SetAccountNameResponse;
+import com.camunda.bpm.example.spring.soap.start.v1.StatusType;
+/**
+ * 
+ * Test various SOAP calls
+ * 
+ */
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations={ "classpath:/spring/engine-test.xml", "classpath:/spring/beans.xml", "classpath:/spring/beans-test.xml"})
+public class BankAccountProcessTest {
+
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
+
+  @Value("${secret}")
+  private String secret;
+
+  @Autowired
+  @Qualifier("bankAccountServiceClient")
+  protected BankAccountServicePortType client;
+
+  @Test
+  public void call_service_returnSuccess() throws Exception {

```

Comment:
callServiceReturnSuccess would be better instead of '_'.Kindly change the naming for the other test cases as well.